### PR TITLE
Dockerfile: added legacy 5.6 variant

### PR DESCRIPTION
**Pull request by bryanlatten**
- Alphabetized most mods installed
- Cleaned up README, now reflects upstream maintainer's 7.1 support status
- Made NewRelic components explicit, if/when we downgrade to an older version for stability

### CODE REVIEW

**Comment by ko-be** [comment]
👍  except tests failed

File: .travis.yml [hunk 1]
```diff
@@ -15,4 +15,4 @@ before_install:
 script:
 - docker-compose build ubuntu
 - docker-compose build edge
-- docker-compose build alpine
```
Comment (bryanlatten) [inline .travis.yml:18]: moving alpine out of the critical path

**Comment by ko-be** [comment]
👍

**Comment by mannytoledo** [comment]
👍

**Pull request merged**